src/Cassandra/Mapping/AppliedInfo.cs

```diff
@@ -15,6 +15,11 @@
 //
 
 using System.Linq;
+using System.Threading.Tasks;
+
+#if !NETSTANDARD2_1_OR_GREATER
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
Is this because `FromRowSetAsync` is marked as async but never awaits if it's NETFRAMEWORK?
I'd prefer to have `FromRowSet` and `FromRowSetAsync` separate so someone doesn't get tricked in the future into using this assuming it's always async when it's actually not in the case of NETFRAMEWORK

Reply:
It will make us have more conditions in the places where this method is being called but hopefully they aren't that many

Reply:
> Is this because FromRowSetAsync is marked as async but never awaits if it's NETFRAMEWORK?

Exactly.

> I'd prefer to have FromRowSet and FromRowSetAsync separate

Done